src/sentry/static/sentry/app/views/settings/components/forms/rangeField.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import InputField from 'app/views/settings/components/forms/inputField';
+import RangeSlider from 'app/views/settings/components/forms/controls/rangeSlider';
+
+type DefaultProps = {
+  formatMessageValue?: false | Function;
+};
+
+type DisabledFunction = (props: any) => boolean;
```

Comment:
Can we type this better? If I understand it correctly, we are passing `otherProps` so basically `RangeField `'s props without `formatMessageValue `